neqo-transport/src/frame.rs

```diff
@@ -594,9 +594,13 @@ impl Frame {
             }),
             FRAME_TYPE_CRYPTO => {
                 let o = dv!(dec);
+                let data = d!(dec.decode_vvec());
+                if o + u64::try_from(data.len()).unwrap() > ((1 << 62) - 1) {
```

Comment:
BTW, I have confirmed that this addition can't overflow as both operands are guaranteed to be < 2^62.

Reply:
:)